src/modules/users/UsersPage.cpp

```diff
@@ -447,30 +464,39 @@ UsersPage::onRootPasswordTextChanged( const QString& )
     }
     else
     {
-        bool ok = true;
         for ( auto pc : m_passwordChecks )
         {
             QString s = pc.filter( pw1 );
+
+            if ( ! ui->checkBoxValidatePassword->isChecked() &&
+                pc.getId() == PasswordCheck::LIBPWQ )
+                continue;
+
             if ( !s.isEmpty() )
             {
                 labelError( ui->labelRootPassword, ui->labelRootPasswordError, s );
-                ok = false;
                 m_readyRootPassword = false;
                 break;
             }
         }
 
-        if ( ok )
+        if ( m_readyRootPassword )
         {
             labelOk( ui->labelRootPassword, ui->labelRootPasswordError );
-            m_readyRootPassword = true;
         }
     }
 
     emit checkReady( isReady() );
 }
 
 
+void
+UsersPage::setValidatePasswordDefault( bool checked )
```

Comment:
the "Default" part here seems weird -- you're actually doing the setting, not changing a default somewhere.